lighty-modules/lighty-gnmi/lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/rcgnmi/GnmiGetITTest.java

```diff
@@ -186,13 +181,11 @@ public void getIncorrectListEntryTest() throws InterruptedException, IOException
     }
 
     @Test
-    public void getNonExistingDataTest() throws InterruptedException, IOException, JSONException {
+    public void getNonExistingDataTest() throws InterruptedException, IOException {
         //assert error for request to non existing interfacesincorrect container
         final HttpResponse<String> getOcInterfacesContainerWrongResponse =
             sendGetRequestJSON(INTERFACES_PATH + "incorrect");
-        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, getOcInterfacesContainerWrongResponse.statusCode());
-        JSONAssert.assertEquals(OC_INTERFACES_INCORRECT_ERROR_MESSAGE_EXPECTED,
```

Comment:
Why is required to remove this JSONAssert? Please compare it with new response:

`    private static final String OC_INTERFACES_INCORRECT_ERROR_MESSAGE_EXPECTED = "{\n  \"errors\": {\n\"error\": [\n"
            + "      {\n        \"error-tag\": \"data-missing\",\n \"error-message\": \"Schema for "
            + "'(http://openconfig.net/yang/interfaces?revision=2021-04-06)interfacesincorrect' not found\",\n"
            + "        \"error-type\": \"protocol\"\n      }\n    ]\n}\n}";`

Reply:
It doesn't make sense to test something which is not relevant to gnmi at all, the error message comes from ODL and may change any time, asserting error code is enough

Reply:
Yes, you are correct, make sense thanks.